pcluster/pcluster_stack.py

```diff
@@ -17,14 +17,18 @@
     region_info
 )
 import json
+from pcluster import __version__
 
 class PclusterStack(cdk.Stack):
 
     def __init__(self, scope: cdk.Construct, id: str, **kwargs) -> None:
         super().__init__(scope, id, **kwargs)
 
+        # Version of ParallelCluster for Cloud9
```

Comment:
What happens if someone sets 2.7.0?

Reply:
Fully supported. The Parameter is free-text, so they can type in any valid release number. Invalid release numbers would cause bootstrap errors.

Reply:
I have a change to this code to limit allowed_values to the actual list from pypi.